scripts/seed_connectors.sql

```diff
@@ -14,181 +14,170 @@ declare
   connector_id flowid;
 begin
```

Comment:
There appear to be deeper changes here than just updating the associated metadata fields?

In any case perhaps we can just kill this `seed_connectors.sql`. We have `local/install-connector.sh` for local development, and it's *not* a goal that we have source-control for every connector installed in the production DB, so I don't believe it serves a purpose anymore.